src/Compilers/VisualBasic/Test/Semantic/Binding/BindingCollectionInitializerTests.vb

```diff
@@ -36,14 +36,15 @@ End Class
             CompileAndVerify(source, "Hello World!")
 
             Dim expectedOperationTree = <![CDATA[
-IObjectCreationExpression (Constructor: Sub System.Collections.Generic.List(Of System.String)..ctor()) (OperationKind.ObjectCreationExpression, Type: System.Collections.Generic.List(Of System.String)) (Syntax: 'New List(Of ... lo World!"}')
-  Arguments(0)
-  Initializer: 
-    IObjectOrCollectionInitializerExpression (OperationKind.ObjectOrCollectionInitializerExpression, Type: System.Collections.Generic.List(Of System.String)) (Syntax: 'From {"Hello World!"}')
-      Initializers(1):
-          ICollectionElementInitializerExpression (AddMethod: Sub System.Collections.Generic.List(Of System.String).Add(item As System.String)) (IsDynamic: False) (OperationKind.CollectionElementInitializerExpression, Type: System.Void, IsImplicit) (Syntax: '"Hello World!"')
-            Arguments(1):
-                ILiteralExpression (OperationKind.LiteralExpression, Type: System.String, Constant: "Hello World!") (Syntax: '"Hello World!"')
+ILocalInitializer (OperationKind.LocalInitializer) (Syntax: 'New List(Of ... lo World!"}')
```

Comment:
It looks like there are two nodes associated with this syntax. Why none of them is marked as IsImplicit?